ansible/inventories/sandbox/group_vars/inventory-next/vars.yml

```diff
@@ -29,13 +29,23 @@ inventory_ckan_bucket_name: "{{ inventory_next_ckan_s3_bucket_name }}"
 inventory_ckan_bucket_prefix: "{{ inventory_next_ckan_s3_bucket_prefix }}"
 ckan_site_domain: "{{ inventory_next_ckan_service_url }}"
 
-inventory_app_repo_branch: inventory_ckan_2.8
+inventory_app_repo_branch: inventory-28-with-saml2
```

Comment:
Will merge inventory-28-with-saml2 to inventory_ckan_2.8 after the PR gets deplyed to staging and production. Otherwise staging and production inventory-next will break when this PR is under review.